app/views/admin/legacy/markdown_contents/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<p><%= link_to 'Create new markdown content', new_admin_legacy_markdown_content_path %></p>
```

Comment:
Not sure why the git diff picked this up as a deleted & completely new file, instead of it being moved.